### PR TITLE
More integration testing

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

**Comment by otoolep** [comment]
@corylanou 

All green, multiple local runs, and on Travis.

File: cmd/influxd/server_single_integration_test.go [hunk 1]
```diff
@@ -139,8 +137,10 @@ func Test_ServerSingleIntegration(t *testing.T) {
 
 	u = urlFor(c.BrokerURL(), "query", url.Values{"q": []string{"CREATE RETENTION POLICY bar ON foo DURATION 1h REPLICATION 1 DEFAULT"}})
 
-	resp, err = httpClient.Get(u.String())
+	resp, err = http.Get(u.String())
 	if err != nil {
+		fmt.Println("*********** FAIL -- sleeping")
```
Comment (otoolep) [inline cmd/influxd/server_single_integration_test.go]: Debug code -- removing now.

**Comment by otoolep** [comment]
Debug code removed.

**Comment by corylanou** [comment]
+1  Looks great!

**Pull request merged**